src/main/java/com/google/sps/data/TimeRange.java

```diff
@@ -168,6 +198,67 @@ public static int getTimeInMinutes(int hours, int minutes) {
     return (hours * 60) + minutes;
   }
 
+  public static int getTimeInMinutes(int day, int hours, int minutes){
```

Comment:
Can you add a comment explaining what this does? It's not really clear why you want to convert a day & time into minutes.

Reply:
Added comment (and some other docs clarifying why the function is important). Both constructors for the TimeRange Object take arguments in minutes, so converting to minutes given a day & time makes that easier.
Though as I'm writing this I'm realizing it might make more sense to modify the constructors - I'll talk about it/ask about it during the daily sync.